packages/cli/src/lib/Cli.ts

```diff
@@ -31,6 +31,7 @@ import {merge} from './cmds/merge';
 import {fingerprint} from './cmds/fingerprint';
 // import {play, PlayArgs} from './cmds/play';
```

Comment:
Should this line be removed?

Reply:
No, we are keeping it for now so we can just uncomment when we want to test the feature in the wild. This also lets anyone that wants to clone the repo and try out the feature do so early without too much modification to the code base.